docs/application/dotnet/guides/nui/videoview.md

```diff
@@ -1,58 +1,99 @@
+---
+
 # VideoView
 
-The `VideoView` class is a control for video playback and display. It provides minimum functions for playback (play, pause, stop, forward, and backward). Some options, such as volume, can be controlled through the control properties.
+The [VideoView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.VideoView.html) class is a control for video playback and display. It provides minimum functions for playback (play, pause, stop, forward, and backward). Some options, such as volume, can be controlled through the control properties. NUI supports many kinds of video format, such as `.avi`, `.3gp` and `mp4`.
 
  > **Note**  
- > You must install a video plugin to enable `VideoView`.
+ > The VideoView control does not use any privileges APIs on its own. However, if you use video files in a specific device storage, the application requires privileges to access the storage.
 
-**Figure: VideoView**
 
-![VideoView](./media/dali_videoview.png)
+## Create a VideoView
 
- > **Note**  
- > The VideoView control does not use any privileges APIs on its own. However, if you use video files in a specific device storage, the application requires privileges to access the storage.
+The following basic example shows how to create a `VideoView` object:
 
-In this tutorial, the following subjects are covered:
+**Figure: Basic VideoView**
 
-[VideoView event](#1)<br>
-[Creating a VideoView](#2)<br>
-[VideoView Properties](#3)<br>
+![Basic VideoView](./media/basicVideoview.png)
 
-<a name="1"></a>
-## VideoView event
+1.  To use the `VideoView` class, add the following namespaces:
 
-The following table lists the basic signal provided by the `VideoView` class.
+    ```cs
+    using Tizen.NUI;
+    using Tizen.NUI.BaseComponents;
+    ```	    
+  
+2.   The video file is assumed to be in the resources directory. Create an instance of the `VideoView` class and use the `ResourceUrl` property to pass the path to the video file. Set `WidthResizePolicy` and `HeightResizePolicy` to make video use full window width and adjust height to width. In the end invoke `Play()` method to start video:
 
-**Table: VideoView input signals**
+      ```cs
+      VideoView player = new VideoView();
+      player.ResourceUrl = DirectoryInfo.Resource + "sample.mp4";
+      player.WidthResizePolicy = ResizePolicyType.FillToParent;
+      player.HeightResizePolicy = ResizePolicyType.DimensionDependency;
+      player.Play();
 
-| Input signal  | Description                                 |
-| ------------- | ------------------------------------------- |
-| `Finished`    | Emitted when a video playback is finished.  |
+      Window.Instance.Add(player);
+      ```
 
-The `VideoView` class provides the `Finished`, which is emitted when the video playback is finished. The related callback can support some basic actions.
+> **Note**  
+> You can set the video file to be played in the `new VideoView()` function, or by modifying `Video` property.
```

Comment:
`Video` property.

why is Video in code tag?

Reply:
Because it is property `player.Video = ...`, similiar to the previous `player.ResourceUrl = ...`

Reply:
@pkalota thanks for claryfing minor change please add the:

 You can set the video file to be played in the `new VideoView()` function, or by modifying the `Video` property.